src/snyk/snykOss/messages/vulnerabilityCount.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ModuleVulnerabilityCount,
+  ModuleVulnerabilityCountSeverity,
+} from '../services/vulnerabilityCount/importedModule';
+
+export const messages = {
+  fetchingVulnerabilities: 'Fetching vulnerabilities...',
+
+  decoratorMessage: (vulnerabilityCount: number): string =>
+    vulnerabilityCount > 1 ? 'vulnerabilities' : 'vulnerability',
+  diagnosticMessage: (module: ModuleVulnerabilityCount): string => {
+    if (!module.count || !module.severityCounts) {
+      return '';
+    }
+
+    let message = `Dependency ${module.name}${module.version ? `@${module.version}` : ''} has `;
+    message += getSeverityCountMessage(
+      [
+        ModuleVulnerabilityCountSeverity.Critical,
+        ModuleVulnerabilityCountSeverity.High,
+        ModuleVulnerabilityCountSeverity.Medium,
+        ModuleVulnerabilityCountSeverity.Low,
+      ],
+      module,
+    );
+    message += module.count > 1 ? ' vulnerabilities.' : ' vulnerability.';
```

Comment:
Don't you usually use a message class for text output?

Reply:
I usually put into `messages` object all the potentially i18n strings. You're right, the logic shouldn't be here, thus moved it out to the actual class.